pennylane_aqt/device.py

```diff
@@ -68,6 +68,7 @@ class AQTDevice(QubitDevice):
         "PauliY": None,
         "PauliZ": None,
         "Hadamard": None,
+        "S": None,
```

Comment:
Realized this gate was needed because QubitDevice uses it in decompositions of certain observables

Reply:
Yep, this is used to measure in the Pauli Y basis if I recall :thinking: